lib/services/dataLake.Analytics/lib/account/dataLakeAnalyticsAccountManagementClient.d.ts

```diff
@@ -56,9 +56,10 @@ declare class DataLakeAnalyticsAccountManagementClient {
     generateClientRequestId: boolean;
 
     // Operation groups
+    firewallRules: operations.FirewallRules;
     storageAccounts: operations.StorageAccounts;
     dataLakeStoreAccounts: operations.DataLakeStoreAccounts;
-    account: operations.Account;
+    accountOperations: operations.AccountOperations;
```

Comment:
@amarzavery this is still an issue it looks like in the AutoRest code gen. Can you take a look?